nsxt/resource_nsxt_policy_group_test.go

```diff
@@ -352,15 +403,22 @@ func testAccNsxtPolicyGroupExists(resourceName string, domainName string) resour
 
 func testAccNsxtPolicyGroupCheckDestroy(state *terraform.State, displayName string, domainName string) error {
 	connector := getPolicyConnector(testAccProvider.Meta().(nsxtClients))
-	nsxClient := domains.NewDefaultGroupsClient(connector)
+
 	for _, rs := range state.RootModule().Resources {
 
 		if rs.Type != "nsxt_policy_group" {
 			continue
 		}
 
+		var err error
 		resourceID := rs.Primary.Attributes["id"]
-		_, err := nsxClient.Get(domainName, resourceID)
+		if isPolicyGlobalManager(testAccProvider.Meta()) {
```

Comment:
I think resourceNsxtPolicyGroupExistsInDomain can be used here?

Reply:
Good catch. Let me fix this.